codalab/server/bundle_manager.py

```diff
@@ -129,13 +137,6 @@ def _stage_bundles(self):
                 bundles_to_fail.append((bundle, str(e)))
                 continue
 
-            missing_uuids = parent_uuids - all_parent_uuids
```

Comment:
Why is this moved up?  In general, can you add a few more comments to your PRs?

Reply:
See my comment about this PR being based on #2795 (this was a change done in #2795)